src/components/02-organisms/Difference/index.js

```diff
@@ -78,7 +86,7 @@ const Difference = ({
         </div>
         <div className="flex-1 flex-col">
           <Typography>{inputAreaTitle}</Typography>
-          <div>
+          {!readonly && <div>
```

Comment:
Is this div really needed?

Reply:
No. This was just following the pattern of how the component was done. I've removed all unnecessary divs now.